eePlumB/2_data_ingestion/re_pull_functions.py

```diff
@@ -46,9 +48,11 @@ def applyScaleFactors_89(image):
     """
     opticalBands = image.select("SR_B.").multiply(0.0000275).add(-0.2)
     thermalBands = image.select("ST_B..").multiply(0.00341802).add(149.0)
```

Comment:
This may be due to my minimal Python skills but just flagging in case this double period in "ST_B.." is a typo. Since it doesn't seem the thermal bands are used downstream maybe this is moot but just a small thing I noticed.

Reply:
Good eye! I could have just written that one out since it's only one band, but it's meant as selecting any band that starts with "ST_B" and is followed by two characters ("..") - alternatives: "ST_B.*" which would select anything that starts with "ST_B", has any character, and then is of any length ("*"). This is all to say, no, it doesn't matter because I don't end up using those bands later, but it *would* work for selecting "ST_B10". :)[ More detail here,](https://developers.google.com/earth-engine/apidocs/ee-image-select) if you are interested at all!

Reply:
Good eye! I could have just written that one out since it's only one band, but it's meant as selecting any band that starts with "ST_B" and is followed by two characters ("..") - alternatives: "ST_B.*" which would select anything that starts with "ST_B", has any character, and then is of any length ("*"). This is all to say, no, it doesn't matter because I don't end up using those bands later, but it *would* work for selecting "ST_B10". :)[ More detail here,](https://developers.google.com/earth-engine/apidocs/ee-image-select) if you are interested at all!

Reply:
Ah got it, thank you! I love learning things in PRs 😄

Reply:
YAY!!!!! Goal has been achieved!!